src/example-types/number.js

```diff
@@ -22,34 +22,40 @@ let rangeFilter = (field, min, max) => ({
 module.exports = {
   hasValue: context => _.isNumber(context.min) || _.isNumber(context.max),
   filter: ({ field, min, max }) => rangeFilter(field, min, max),
-  async result({ field, min, max }, search) {
+  async result({ field, min, max, percentileInterval = 1 }, search) {
     let statisticalResult = await search({
       aggs: {
         range_filter: {
           filter: rangeFilter(field, min, max),
           aggs: {
             statistical: {
-              extended_stats: {
+              stats: {
                 field,
                 missing: 0,
-                sigma: 1,
               },
             },
             all_percentiles: {
               percentiles: {
                 field,
-                percents: [0, 0.5, 99.5, 100],
+                percents: [0, percentileInterval, 100 - percentileInterval, 100],
+                keyed: false
```

Comment:
instead of keyed: false, just do a `F.pickInto` to map into a different named structure